packages/validator/src/plugins/validation/2and3/semantic-validators/paths-ibm.js

```diff
@@ -89,17 +89,17 @@ module.exports.validate = function({ resolvedSpec }, config) {
         });
 
         if (!accountsForAllParameters) {
-          const checkStatus = config.missing_path_parameter;
-          if (checkStatus != 'off') {
-            missingParameters.forEach(name => {
-              messages.addMessage(
-                ['paths', pathName, opName, 'parameters'],
-                `Operation must include a path parameter with name: ${name}.`,
-                checkStatus,
-                'missing_path_parameter'
-              );
-            });
-          }
+          // const checkStatus = config.missing_path_parameter;
+          // if (checkStatus != 'off') {
+          //   missingParameters.forEach(name => {
+          //     messages.addMessage(
+          //       ['paths', pathName, opName, 'parameters'],
+          //       `Operation must include a path parameter with name: ${name}.`,
+          //       checkStatus,
+          //       'missing_path_parameter'
+          //     );
+          //   });
+          // }
```

Comment:
I guess this should be removed, right?

Reply:
yep, I intended to take another pass through the changes in the legacy rule and remove these commented-out sections.  will fix.